bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/DigitalInvoiceScreenTests.kt

```diff
@@ -0,0 +1,199 @@
+        digitalInvoiceScreen.clickArticleSwitch()
+        val isItemDisabled =digitalInvoiceScreen.checkItemIsEnabledFromDigitalScreen()
+        assertEquals(true, isItemDisabled)
+    }
+
+    @Test
+    fun test7_differenceInTotalAmountWithSwitchEnabledOrDisabled() {
+        test5_clickOnItemFromReturnReasonsList()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.checkTotalTitleIsDisplayed()
+        digitalInvoiceScreen.checkTotalPriceIsDisplayed()
+        digitalInvoiceScreen.storeInitialPrice()
+        digitalInvoiceScreen.clickArticleSwitch()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.checkTotalPriceIsDisplayed()
+        digitalInvoiceScreen.storeUpdatedPrice()
+        digitalInvoiceScreen.assertPriceHasChanged()
+        idlingResource.waitForIdle()
+    }
+
+    @Test
+    fun test8_verifyAdditionalChargesOnDigitalInvoice() {
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertOtherChargesDisplayed()
+    }
+
+    @Test
+    fun test9_clickProceedButtonOnDigitalInvoiceScreen() {
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertDigitalInvoiceText()
+        digitalInvoiceScreen.clickProceedButton()
+        extractionScreen.checkTransferSummaryButtonIsClickable()
+    }
+
+    @Test
+    fun test10_clickHelpButtonAndVerifyContentOnHelpScreen() {
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertDigitalInvoiceText()
+        digitalInvoiceScreen.clickHelpButtonOnDigitalInvoiceScreen()
+        digitalInvoiceScreen.verifyHelpTextOnNextScreen()
+        digitalInvoiceScreen.verifyFirstTitleOnHelpScreen()
+    }
+
+    @Test
+    fun test11_clickCancelButtonOnDigitalInvoiceScreen() {
```

Comment:
[Clarification] Same comment as for `test9_`...
I see 2 options here:

- Check if `cancelButton` is `clickable`
- Check if `mainScreenDescriptionTitle` is displayed after clicking the `cancelButton`

Reply:
It has the similar observation as mentioned above.. However, I have changed the title `checkMainScreenTitleIsDisplayedAfterClickOnCancelButton`